src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
+      <div class="stars stars__star"></div>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to improve readability. There should be an empty line between each `div.stars` block.